bookkeeper-server/src/test/java/org/apache/bookkeeper/client/DeferredSyncBookieTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2017 The Apache Software Foundation.
```

Comment:
remove this line. it is an apache project, you shouldn't include "Copyright" line here.

Reply:
sorry, wrong license, done